azext_iot/product/_help.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for Product Certification commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_help():
+    helps["iot product"] = """
+        type: group
+        short-summary: Manage device testing for product certification
```

Comment:
Cool I like that `product certification` or `certification` is everywhere in the descriptions